deploy/cht_sync/values.yaml.template

```diff
@@ -0,0 +1,32 @@
+
+postgres:
+  enabled: true  # if true a postgres service will not be created in this cluster
+  user: ""
+  password: ""
+  db: ""
+  schema: "v1"
+  table: "medic"  
+  host: "postgres" # if postgres is outside the cluster
+  port: 5432 # if postgres is outside the cluster
+
+cht_pipeline_branch_url: "https://github.com/medic/cht-pipeline.git#main"
+
+# values shared by all couchdb instances
+# can be omitted if couchdb instances do not share any values
+couchdb:
+  user: ""
+  dbs: "medic" # space separated list of databases you want to sync e.g "medic medic-sentinel"
+  port: "5984"
+  secure: "true" # "true" if SSL is required
+
+# values for each couchdb instance
+# host and password are required
+# other values can be ommitted if common to all couchdb instances and specified above
+couchdbs: # values for in
```

Comment:
This doesn't really exemplify of how I would add multiple couchdb servers to sync from.

Reply:
can you elaborate on this? what specifically does not exemplify how would you add multiple couchdb servers to sync from?

Reply:
As a reader I'm not imagining how this would look like. 

Is it like: 
```
couchdbs: # values for in
  - host: "host1"
    password: ""
    user: ""
    dbs: "medic medic-sentinel"
    port: "5984"
    secure: "true"

   - host: "host2"
    password: ""
    user: ""
    dbs: "medic medic-users"
    port: "5984"
    secure: "true"
```

I think it would be helpful if we add an example, because these iterator types are weird in yml

Reply:
yes, that's what its supposed to be.
There is a section `couchdb` for shared values, and `couchdbs` for a list of values which are not shared.
which makes it a little more complicated, but otherwise when user,dbs,port and secure are NOT different, like for MoH Kenya, have to copy them 47 times, and don't like copy/pasting that much.

added an example to values.yaml.template

```
couchdbs:
  - host: "host1" # required for all couchdb instances
    password: "" # required for all couchdb instances
  - host: "host2"
    password: ""
  - host: "host3"
    password: ""
    user: "user2" # required if different than above
    dbs: "medic medic_sentinel" # required if different than above
    port: "5984" # required if different than above
    secure: "true" # required if different than above
```